tests/app/questionnaire/test_questionnaire_store_updater.py

```diff
@@ -1157,7 +1156,9 @@ def test_dependent_sections_completed_dependant_blocks_removed_and_status_update
     }
 
     assert dependent_block_id in progress_store.get_completed_block_ids(
-        section_id=dependent_section_key[0], list_item_id=dependent_section_key[1]
+        section_key=SectionKey(
+            section_id=dependent_section_key[0], list_item_id=dependent_section_key[1]
```

Comment:
I think this could have been `SectionKey(*dependent_section_key)` same for others in the file